edx_exams/apps/lti/views.py

```diff
@@ -183,6 +184,8 @@ def start_proctoring(request, attempt_id):
         message_type='LtiStartProctoring',
         proctoring_launch_data=proctoring_launch_data,
         context_id=exam.course_id,
+        context_title=settings.LEARNING_MICROFRONTEND_URL,
```

Comment:
Question: I'm using context title to pass along the MFE url but let me know if there is a better way to pass along in launch data so that it can be used in xblock lti consumer.

Reply:
If it's already in settings, you don't need to pass it to xblock-lti-consumer. That plugin will also have access to the same settings